src/app/i18n/language-switcher/switcher.js

```diff
@@ -1,13 +1,19 @@
 
 class LanguageSwitcherController {
   /** @ngInject */
-  constructor($translate) {
+  constructor($document, $translate) {
     this.$translate = $translate;
+    this.$document = $document;
   }
 
   onSwitcherClicked() {
     const currentLang = this.$translate.use();
     const newLang = currentLang === 'en' ? 'mn' : 'en';
+    if (newLang === 'mn') {
+      this.$document.find('html').addClass('cyrillic');
+    } else {
+      this.$document.find('html').removeClass('cyrillic');
```

Comment:
`toggleClass` takes an argument for whether the class should be applied or not, so you could do `this.$document.find('html').toggleClass('cyrillic', newLang === 'mn')`. Comes across a bit obtuse though, especially with the stretched meaning of 'toggle'.